lib/matplotlib/widgets.py

```diff
@@ -1066,12 +1072,22 @@ def _rendercursor(self):
 
         text = self.text_disp.get_text()  # Save value before overwriting it.
         widthtext = text[:self.cursor_index]
+
+        bb_text = self.text_disp.get_window_extent()
         self.text_disp.set_text(widthtext or ",")
-        bb = self.text_disp.get_window_extent()
-        if not widthtext:  # Use the comma for the height, but keep width to 0.
-            bb.x1 = bb.x0
+        bb_widthtext = self.text_disp.get_window_extent()
+
+        if bb_text.y0 == bb_text.y1:  # Restoring the height if no text.
```

Comment:
This seems to need testing.

Reply:
It should be fine now but codecov is not starting so I'm not sure